apps/reprocessor-mediator-microfrontend/src/components/StepOne.tsx

```diff
@@ -0,0 +1,217 @@
+import { useState } from "react";
+import {
+  Grid,
+  Typography,
+  Button,
+  TextField,
+  MenuItem,
+  Card,
+  CardContent,
+  Select,
+  Checkbox,
+  ListItemText,
+  FormControl,
+  InputLabel,
+} from "@mui/material";
+
+const availableResources = [
```

Comment:
can you add MedicationStatement, ServiceRequest, MedicationDispense, QuestionnaireResponse, Procedure, DiagnosticReport

Reply:
Sure.